### ISSUE TITLE
Account general settings back button is missing accessible label

### ISSUE THREAD

**Issue by kewisch** [thunderbird/thunderbird-android#9457]
The new settings page has a back button that isn't labelled like our others.

STR:

1. Go to Settings > demo@example.com > General Settings
2. Observe the "back" button 

Expected:

* Back button should have text "Navigate up" like all our other back buttons

Actual

* Back button has id `SubtitleTopAppBarBackButton` but no text

**Comment by shivam-sharma-2000** [comment]
I’d like to work on it. Could you please assign it to me?
